demos/textfield.html

```diff
@@ -218,54 +295,76 @@
         </div>
         <div>
-          <input id="multi-dark-theme" type="checkbox">
-          <label for="multi-dark-theme">Dark Theme</label>
+          <input id="textarea-dark-theme" type="checkbox">
+          <label for="textarea-dark-theme">Dark Theme</label>
         </div>
         <div>
-          <input id="multi-required" type="checkbox">
-          <label for="multi-required">Required</label>
+          <input id="textarea-required" type="checkbox">
+          <label for="textarea-required">Required</label>
         </div>
       </section>
+
       <section class="example">
-        <h2>Multi-line Textfields - CSS Only</h2>
-        <label for="css-only-multiline" style="margin-bottom:4px;">About you:</label>
-        <div class="mdc-textfield mdc-textfield--multiline" data-demo-no-auto-js>
-          <textarea class="mdc-textfield__input"
-                    id="css-only-multiline"
-                    rows="8" cols="40"
-                    placeholder="Tell the world something about yourself!"></textarea>
+        <h2>Textarea with Icon</h2>
+        <section id="demo-textfield-textarea-with-icon-wrapper">
+          <div class="mdc-textfield mdc-textfield--textarea mdc-textfield--textarea-with-icon">
+            <textarea id="textarea-icon" class="mdc-textfield__input" rows="8" cols="40"></textarea>
+            <label for="textarea-icon" class="mdc-textfield__label">Textarea Label</label>
+            <i class="material-icons mdc-textfield__icon" tabindex="0">cancel</i>
+          </div>
+        </section>
+        <div>
+          <input id="textarea-icon-rtl" type="checkbox">
+          <label for="textarea-icon-rtl">RTL</label>
+        </div>
+        <div>
+          <input id="textarea-icon-dark-theme" type="checkbox">
+          <label for="textarea-icon-dark-theme">Dark Theme</label>
         </div>
       </section>
+
+      <section class="example">
+        <h2>CSS Only Textarea</h2>
+        <section>
+          <div class="mdc-textfield mdc-textfield--textarea" data-demo-no-auto-js>
+            <textarea id="textarea-css-only" class="mdc-textfield__input" rows="8" cols="40" placeholder="Enter something about yourself"></textarea>
+          </div>
+        </section>
+      </section>
+
       <section class="example">
-        <h2>Full-Width Textfields</h2>
+        <h2>Full-Width Textfield and Textarea</h2>
         <div id="demo-fullwidth-wrapper">
           <div class="mdc-textfield mdc-textfield--fullwidth" data-demo-no-auto-js>
             <input class="mdc-textfield__input" type="text" placeholder="Subject" aria-label="Subject">
           </div>
-          <div class="mdc-textfield mdc-textfield--multiline mdc-textfield--fullwidth" data-demo-no-auto-js>
-            <textarea class="mdc-textfield__input" placeholder="Message" rows="8" cols="40" aria-label="Message"></textarea>
+          <div class="mdc-textfield mdc-textfield--textarea mdc-textfield--fullwidth full-width-textarea-example">
```

Comment:
I think you're right. Since the structure of the demos is mostly up to us, there just wasn't any design guidance on this. I added the bottom line and removed `data-demo-no-auto-js` so it matches the `textarea` more.